hypershift-operator/controllers/scheduler/aro/aro_scheduler.go

```diff
@@ -0,0 +1,157 @@
+package aro
```

Comment:
nit: "aro" and "scheduler" are already part of the package name. I would name the file something like "controller.go" or "reconciler.go"

Reply:
tagging on this... Alberto had me removing some of the ARO verbiage in favor of something like "managed Azure" in previous ARO HCP PRs. If you search around our codebase, you'll see where we already transitioned a lot to that type of language.

Reply:
This Repo allocates a project which supports multiple products. There's anything about this code that makes it aro specific. I see it as an agnostic building block that we might choose to leverage in e.g. self-hosted azure. It's a runtime choice to tie it with any particular product. My suggestion is we capture this as semantics when naming things.
With that in mind It would makes sense to me that we name everything here after "Azure". Then, at runtime, we choose to run this controller if IsAroHCP